src/components/Card/styled.js

```diff
@@ -1,9 +1,12 @@
 import styled from 'styled-components';
 
-const StyledCard = styled.section`
+const StyledCard = styled.li`
+	position: relative;
```

Comment:
I would like to suggest to use flex-box for your Cards instead of positioning your button with absolute, this is a way usual for your use case. 
As a little reminder: 
An absolutely positioned element no longer exists in the normal document flow. In some cases that's a perfect and kind of magic tool, but in this case it could lead you into weird bugs. 🤯

Reply:
Thanks a lot for this advice  😃